src/Standards/Squiz/Tests/Formatting/OperatorBracketUnitTest.1.inc

```diff
@@ -201,3 +201,7 @@ match ($a) {
 };
 
 $cntPages = ceil(count($items) / parent::ON_PAGE);
+
+$nsRelative = namespace\doSomething($items + 10);
+$nsRelative = Partially\qualified($items + 10);
+$nsRelative = \Fully\qualified($items + 10);
```

Comment:
I believe the variable for the different test cases should be different as they are not all about relative namespaces.